test/CoWSwapOnchainOrders/CoWSwapOnchainOrdersExposed.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.8;
+
+import "../../src/mixins/CoWSwapOnchainOrders.sol";
+
+/// @dev Wrapper that exposes internal funcions of CoWSwapOnchainOrders.
+contract CoWSwapOnchainOrdersExposed is CoWSwapOnchainOrders {
```

Comment:
Did you wrote this contract, because you actually wanted to implement CoWSwapOnchainOrders as library?

Reply:
Indeed, if `CoWSwapOnchainOrders` had been a library then this contract wouldn't have been needed.
It's just a way to expose internal functions and make them testable.